common/transport/mqtt/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "azure-iot-mqtt-base",
+  "version": "1.1.12",
```

Comment:
>"1.1.12" [](start = 13, length = 8)

we have a release at the end of the week so we need to get this right, right now. we need to double check if the interface is compatible with the previous version, or bump it to 2.0.0

Reply:
You know, even if the interface looks compatible, I don't want to support much older versions of the SDK with much newer versions of this file.  I'd rather just bump to 2.0.0 to show that this is a brand new thing that shouldn't be intermixed with old things.

---
In reply to: [142812748](https://github.com/Azure/azure-iot-sdk-node/pull/101#discussion_r142812748) [](ancestors = 142812748)